html/install.php

```diff
@@ -354,64 +354,35 @@
       <div class="col-md-6">
 <?php
 
-// Create the config file we will write or display
-$config_file = <<<"EOD"
-## Have a look in defaults.inc.php for examples of settings you can set here. DO NOT EDIT defaults.inc.php!
+// Create the .env file we will write or display
+$app_key = trim(`php artisan key:generate --show --no-ansi --no-interaction`);
+$node_id = uniqid();
+$dot_env = <<<"EOD"
+APP_KEY=$app_key
+NODE_ID=$node_id
 
-### Database config
-\$config\['db_host'\] = '$dbhost';
-\$config\['db_port'\] = '$dbport';
-\$config\['db_user'\] = '$dbuser';
-\$config\['db_pass'\] = '$dbpass';
-\$config\['db_name'\] = '$dbname';
-\$config\['db_socket'\] = '$dbsocket';
+DB_HOST=$dbhost
+DB_DATABASE=$dbname
+DB_USERNAME=$dbuser
+DB_PASSWORD="$dbpass"
 
-// This is the user LibreNMS will run as
```

Comment:
Should we still not create the config.php file (or generate and show it at least)? It provides some good standards people can update.